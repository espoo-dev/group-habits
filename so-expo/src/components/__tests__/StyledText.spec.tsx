import { render } from '@testing-library/react-native';
import * as React from 'react';

import { MonoText } from '../StyledText';

function setup() {
  return render(<MonoText>Snapshot</MonoText>)
}

it(`renders correctly`, () => {
  const tree = setup().toJSON()

  expect(tree).toMatchSnapshot();
});
