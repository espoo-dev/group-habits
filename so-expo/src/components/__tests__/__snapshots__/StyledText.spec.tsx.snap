// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    [
      {
        "color": "#1A54F8",
      },
      [
        undefined,
        {
          "fontFamily": "space-mono",
        },
      ],
    ]
  }
>
  Snapshot
</Text>
`;
